internal/statemachine/classic_test.go

```diff
@@ -3011,19 +3010,19 @@ func TestSuccessfulClassicRun(t *testing.T) {
 
 	artifacts := map[string]string{
 		"pc-amd64.img":            "DOS/MBR boot sector",
-		"pc-amd64.qcow2":          "QEMU QCOW",
+		"pc-amd64.qcow2":          "QEMU QCOW Image (v3)",
```

Comment:
I suppose this is enough of an indication that our qcow2 image is sparse, right?

Reply:
See my other comment but that is the only way I found to check the `compat=1.1` option was used without parsing the resulting qcow2 images. I also thought of using `qemu-img info` but comparing the result between V2 and V3 qcow images was not really significant.